strawberryfields/cli/__init__.py

```diff
@@ -0,0 +1,219 @@
+        "--local", "-l", action="store_true", help="Create the configure for the project."
+    )
+
+    # Adding the input subparser
+    run_parser = subparsers.add_parser("run", help="Run a blackbird script.")
+    run_parser.add_argument(
+        "input", type=str, help="The input blackbird script to run.",
+    )
+    run_parser.set_defaults(func=run_blackbird_script)
+    run_parser.add_argument(
+        "--output",
+        "-o",
+        help="Path to the output file, where the results of the program will be written (stdout by default).",
+    )
+
+    return parser
+
+
+def configure(args):
+    """An auxiliary function for configuring the API connection to the Xanadu
+    cloud platform.
+
+    Supports configuration by:
+    * only specifying the token and using default configuration options;
+    * specifying every configuration option one by one.
+
+    Related arguments:
+    * token: the authentication token to use
+    * local: whether or not to create the configuration file locally
+
+    Args:
+        args (ArgumentParser): arguments that were specified on the command
+            line stored as attributes in an argument parser object
+    """
+    if args.token:
+        kwargs = {"authentication_token": args.token}
+    else:
+        kwargs = configure_everything()
+
+    if args.local:
+        store_account(**kwargs, location="local")
+    else:
+        store_account(**kwargs)
+
+
+def ping():
+    """Tests the connection to the remote backend."""
+    if Connection().ping():
+        sys.stdout.write("You have successfully authenticated to the platform!\n")
+    else:
+        sys.stdout.write("There was a problem when authenticating to the platform!\n")
+    sys.exit()
+
+
+def configure_everything():
+    """Provides an interactive selection wizard on the command line to
+    configure every option for the API connection.
+
+    Default configuration options are provided as defaults to the user.
+    Configuration options as detailed in :doc:`/introduction/configuration`.
+
+    Returns:
+        dict[str, Union[str, bool, int]]: the configuration options
+    """
+    default_config = create_config()["api"]
+
+    authentication_token = (
+        input(PROMPTS["authentication_token"].format(default_config["authentication_token"]))
+    )
+
+    if not authentication_token:
+        sys.stdout.write("No authentication token was provided, please configure again.")
+        sys.exit()
+
+    hostname = (
+        input(PROMPTS["hostname"].format(default_config["hostname"])) or default_config["hostname"]
+    )
+
+    ssl_default = "y" if default_config["use_ssl"] else "n"
+    ssl_input = input(PROMPTS["use_ssl"].format(ssl_default)) or ssl_default
+    use_ssl = ssl_input.upper() == "Y"
+
+    port = input(PROMPTS["port"].format(default_config["port"])) or default_config["port"]
+
+    kwargs = {
+        "authentication_token": authentication_token,
+        "hostname": hostname,
+        "use_ssl": use_ssl,
```

Comment:
Should this be `"use_ssl": True`, rather than `"use_ssl": "Y"`? I thought this was a boolean property

Reply:
Never mind, I see my mistake!

Reply:
Right! It is, after all, a boolean as a result of `ssl_input.upper() == "Y"`.

This is merely used such that the user can see `[y]` in the wizard and that `True` would be stored in case they hit any of `y`, `Y` or simply enter (this results in the empty string being parsed `''` and for this reason, is the `or ssl_default` part of the statement needed in `ssl_input = input(PROMPTS["use_ssl"].format(ssl_default)) or ssl_default`). Let me know, however, if you think this could be simplified and/or made more intuitive in the code or in the interaction